multinet/db.py

```diff
@@ -36,6 +36,7 @@
     host=os.environ.get("ARANGO_HOST", "localhost"),
     port=int(os.environ.get("ARANGO_PORT", "8529")),
 )
+RESTRICTED_KEYS = {"_rev", "_id"}
```

Comment:
Does the linter complain if you use lowercase for this?

Reply:
No, I can change it.